### PR TITLE
Implements a Firebase storage system

### PR DESCRIPTION
**Pull request by RafaelCosman**

### CODE REVIEW

**Comment by RafaelCosman** [comment]
This pull request needs to pass ```storage_test.js``` before it can be considered.

File: lib/firebase_storage.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/*
```
Comment (anonrig) [inline lib/firebase_storage.js]: What do you think about JSDoc? http://usejsdoc.org/

Comment (RafaelCosman) [inline lib/firebase_storage.js]: @anonrig looks useful! I assume that you don't mean just for this file but for the whole codebase, right?

Comment (RafaelCosman) [inline lib/firebase_storage.js]: If that's the case, would you please either submit a pull request or an issue? Thanks!

Comment (anonrig) [inline lib/firebase_storage.js]: Thats the case. Will submit a pull request!

**Pull request merged**